build/.vsts-ci.yml

```diff
@@ -2,8 +2,37 @@ name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Re
 variables:
   runCodesignValidationInjection: false
 resources:
-- repo: self
-  clean: true
+  containers:
```

Comment:
`linux_back_compat.yml` and `longhaul.vsts-ci.yml` both still reference `_PREVIEW_VSTS_DOCKER_IMAGE` which has otherwise been removed from this file.

Is this OK or also needs to be fixed?

Reply:
Anywhere we're referencing the `_PREVIEW_VSTS_DOCKER_IMAGE` should probably be updated. If you have others that you know of let me know. I'll try and work my way through the others. I know I have to work to c-utility and our submodules as well.

Reply:
I just did a findstr on this and the places I called out above are only places in this repo I found doing this.

Thanks for taking care of this btw!  Once you update those places & verify I'll be happy to sign off.

Reply:
Weird my find all and replace in VSCode isn't picking them up....okay let me try something else.